src/Akka.Hosting/LoggerConfigBuilder.cs

```diff
@@ -83,6 +84,17 @@ public LoggerConfigBuilder ClearLoggers()
             _loggers.Add(logger);
             return this;
         }
+        
+        /// <summary>
+        /// Sets the formatter used by the logger
+        /// </summary>
+        public LoggerConfigBuilder WithDefaultLogMessageFormatter<T>() where T: ILogMessageFormatter
```

Comment:
Added a type-safe API for setting the default `ILogMessageFormatter` for the `ActorSystem`.